src/styles/cards.scss

```diff
@@ -0,0 +1,119 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  max-width: $card_size;
+  padding: 32px 16px 16px;
+  border: 1px solid $grey_color;
+  border-radius: 5px;
+  transition: all $transition_time;
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    padding-inline: 3px;
+    margin-bottom: 40px;
+  }
+
+  &__info {
+    display: flex;
+    flex-direction: column;
+
+    &--title {
+      color: $dark_blue_color;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      justify-content: center;
+      margin-top: 0;
+      margin-bottom: 4px;
+      height: 36px;
```

Comment:
```suggestion

```
The height of the element should depend on the content

Reply:
Done